server/src/test/java/org/elasticsearch/rest/RestControllerTests.java

```diff
@@ -103,8 +107,13 @@ public void setup() {
         httpServerTransport.start();
     }
 
+    @After
+    public void teardown() {
+        client.close();
```

Comment:
```suggestion
        IOUtils.close(client);
```

This allows us to guard in case the client is null to avoid unnecessary exceptions in case of odd setup failure. The `IOUtils` class I am referencing is from package `org.elasticsearch.core.internal.io`